core/stores/sqlx/stmt.go

```diff
@@ -34,7 +36,70 @@ func SetSlowThreshold(threshold time.Duration) {
 	slowThreshold.Set(threshold)
 }
 
+// FormatSql expand slice or array arguments
+func FormatSql(query string, args ...interface{}) (string, []interface{}) {
```

Comment:
Why FormatSql need to be exported?

Reply:
where？

Reply:
> where?

At present, because there is no such method, our projects are all written in the public library to provide external calls. If there is no need to export when integrated into `go-zero`, then I can change it